\hypertarget{_call_method_exception_8cs}{}\doxysection{C\+:/\+Users/korye/source/repos/csla/\+Source/\+Csla/\+Reflection/\+Call\+Method\+Exception.cs File Reference}
\label{_call_method_exception_8cs}\index{C:/Users/korye/source/repos/csla/Source/Csla/Reflection/CallMethodException.cs@{C:/Users/korye/source/repos/csla/Source/Csla/Reflection/CallMethodException.cs}}
\doxysubsection*{Classes}
\begin{DoxyCompactItemize}
\item 
class \mbox{\hyperlink{class_csla_1_1_reflection_1_1_call_method_exception}{Csla.\+Reflection.\+Call\+Method\+Exception}}
\begin{DoxyCompactList}\small\item\em This exception is returned from the Call\+Method method in the server-\/side \mbox{\hyperlink{class_csla_1_1_data_portal}{Data\+Portal}} and contains the exception thrown by the underlying business object method that was being invoked. \end{DoxyCompactList}\end{DoxyCompactItemize}
\doxysubsection*{Namespaces}
\begin{DoxyCompactItemize}
\item 
namespace \mbox{\hyperlink{namespace_csla}{Csla}}
\item 
namespace \mbox{\hyperlink{namespace_csla_1_1_reflection}{Csla.\+Reflection}}
\end{DoxyCompactItemize}
